apps/prairielearn/src/middlewares/clientFingerprint.ts

```diff
@@ -52,3 +52,31 @@ export default asyncHandler(async (req, res, next) => {
   res.locals.client_fingerprint_id = client_fingerprint_id;
   next();
 });
+
+export async function getClientFingerprintId(req: any, res: any) {
```

Comment:
There are `Request` and `Response` types in the `express` package that you can use for the arguments here.

Reply:
Added use of `Request` and `Response` types in commit [916c5a5](https://github.com/PrairieLearn/PrairieLearn/pull/9087/commits/916c5a53cfa2b70e94b9b2dc43be9bdc3529fd6f).